### PR TITLE
feat(widget-space): Space and WMM events update

### PR DESCRIPTION
**Pull request by bzang**
Update docs to add new eventing, and also add ability to pass eventNames as props.

### CODE REVIEW

File: packages/node_modules/@ciscospark/redux-module-media/src/actions.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export function hangupCall(call) {
   return (dispatch) => {
     // Don't update call states after hangup
     call.off();
-    call.hangup().then(() => {
+    return call.hangup().then(() => {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/redux-module-media/src/actions.js]: Why is this in a docs update PR?

File: packages/node_modules/@ciscospark/widget-meet/src/constants.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-meet/src/constants.js]: Wrong PR?

**Comment by adamweeks** [comment]
Did we want to do the `.activity` here instead of `initialActivity` or will that be later? Probably should decide on it before the final release of the docs

**Comment by bzang** [comment]
That requires a more extensive code change than we probably want at this point. Was gonna wait until the implementation actually happens before updating the docs to make sure that the docs always represent the actual code.

**Pull request merged**